plugins/rss/lang/en.js

```diff
@@ -74,6 +74,8 @@
  theUILang.rssAllFiters			= "All filters";
  theUILang.rssUpdateInterval		= "Update interval";
  theUILang.rssShowErrorsDelayed		= "Delay RSS error notification until the log tab or RSS feed is visible";
+ theUILang.rssSanitizationLevel		= "Show external RSS content";
+ theUILang.rssSanitizationLevels	= {RESTRICTED: "Restricted (no links or images)", BASIC: "Basic (no images)", RELAXED: "Relaxed"};
```

Comment:
I disagree that we require three sanitation levels here. I think `Secure` and `Insecure` should be fine. Stronger terminology also allows the user to make a more education decision about this functionality.